turbopack/crates/turbo-tasks-backend/src/backend/mod.rs

```diff
@@ -0,0 +1,1086 @@
+    }
+
+    fn try_read_task_cell_untracked(
+        &self,
+        task_id: TaskId,
+        cell: CellId,
+        turbo_tasks: &dyn TurboTasksBackendApi<Self>,
+    ) -> Result<Result<TypedCellContent, EventListener>> {
+        self.try_read_task_cell(task_id, None, cell, turbo_tasks)
+    }
+
+    fn try_read_own_task_cell_untracked(
+        &self,
+        task_id: TaskId,
+        cell: CellId,
+        turbo_tasks: &dyn TurboTasksBackendApi<Self>,
+    ) -> Result<TypedCellContent> {
+        let ctx = self.execute_context(turbo_tasks);
+        let task = ctx.task(task_id);
+        if let Some(content) = get!(task, CellData { cell }) {
+            Ok(CellContent(Some(content.clone())).into_typed(cell.type_id))
+        } else {
+            Ok(CellContent(None).into_typed(cell.type_id))
+        }
+    }
+
+    fn read_task_collectibles(
+        &self,
+        _: TaskId,
+        _: TraitTypeId,
+        _: TaskId,
+        _: &dyn TurboTasksBackendApi<Self>,
+    ) -> AutoMap<RawVc, i32, BuildHasherDefault<FxHasher>, 1> {
+        todo!()
+    }
+
+    fn emit_collectible(
+        &self,
+        _: TraitTypeId,
+        _: RawVc,
+        _: TaskId,
+        _: &dyn TurboTasksBackendApi<Self>,
+    ) {
+        todo!()
+    }
+
+    fn unemit_collectible(
+        &self,
+        _: TraitTypeId,
+        _: RawVc,
+        _: u32,
+        _: TaskId,
+        _: &dyn TurboTasksBackendApi<Self>,
+    ) {
+        todo!()
+    }
+
+    fn update_task_cell(
+        &self,
+        task_id: TaskId,
+        cell: CellId,
+        content: CellContent,
+        turbo_tasks: &dyn TurboTasksBackendApi<Self>,
+    ) {
+        operation::UpdateCellOperation::run(
+            task_id,
+            cell,
+            content,
+            self.execute_context(turbo_tasks),
+        );
+    }
+
+    fn connect_task(
+        &self,
+        task: TaskId,
+        parent_task: TaskId,
+        turbo_tasks: &dyn TurboTasksBackendApi<Self>,
+    ) {
+        ConnectChildOperation::run(parent_task, task, self.execute_context(turbo_tasks));
+    }
+
+    fn create_transient_task(
+        &self,
+        task_type: TransientTaskType,
+        _turbo_tasks: &dyn TurboTasksBackendApi<Self>,
+    ) -> TaskId {
+        let task_id = self.transient_task_id_factory.get();
+        let root_type = match task_type {
+            TransientTaskType::Root(_) => ActiveType::RootTask,
+            TransientTaskType::Once(_) => ActiveType::OnceTask,
+        };
+        self.transient_tasks.insert(
+            task_id,
+            Arc::new(match task_type {
+                TransientTaskType::Root(f) => TransientTask::Root(f),
+                TransientTaskType::Once(f) => TransientTask::Once(Mutex::new(Some(f))),
+            }),
+        );
+        {
+            let mut task = self.storage.access_mut(task_id);
+            let _ = task.add(CachedDataItem::AggregationNumber {
```

Comment:
Are the `let _ =`s really needed here? It looks like the return type of `InnerStorage::add` is bool, which shouldn't be `#[must_use]`?